src/server/api/routes/profile.router.js

```diff
@@ -31,4 +31,27 @@ router.get('/student', (req, res, next) => {
     .catch(next);
 });
 
+/**
+ * @swagger
+ * /profile/admin:
+ *  get:
+ *    tags:
+ *    - Profile
+ *    summary: Get all admins profile
+ *    description:
+ *      Will return all admins profile.
+ *    produces: application/json
+ *    responses:
+ *      200:
+ *        description: Successful request
+ *      5XX:
+ *        description: Unexpected error.
+ */
+
+router.get('/admin', (req, res, next) => {
+  profileController
+    .getAdminsProfile()
```

Comment:
This is not _all_ admins but just the logged in one. It's _your own_ profile if you log in.

Reply:
but how do  I get logged in info in backend?

Reply:
You can read about it https://github.com/HackYourFuture-CPH/web-dev-rooms/blob/develop/Authentication.md

Reply:
thanks